python_modules/libraries/dagster-gcp/dagster_gcp/bigquery/assets.py

```diff
@@ -0,0 +1,37 @@
+import pandas as pd
+from dagster import DataVersion, observable_source_asset
+
+from .resources import BigQueryResource
+
+"""
+Running the following query in bigquery gets you the last updated timestamp:
```

Comment:
Do we know what timezone this returns in? Can we safely assume it's always UTC?

Reply:
the field we're operating on is already an int timestamp, timestamp_millis just converts to a utc date translateable timestamp. See here: https://cloud.google.com/bigquery/docs/reference/standard-sql/timestamp_functions#timestamp_millis